.github/workflows/test-spec.yml

```diff
@@ -32,13 +32,10 @@ jobs:
       - name: Download spec tests
         run: yarn download-test-cases
         working-directory: packages/spec-test-runner
-      - name: Spec min
-        run: yarn test:spec-min
+      - name: Spec tests
+        run: yarn test:spec
         working-directory: packages/spec-test-runner
-      - name: Spec fast
-        run: yarn test:spec-fast
-        working-directory: packages/spec-test-runner
-      - name: Spec main
-        run: yarn test:spec-main
```

Comment:
Why un-group this different runs?

Reply:
Good question
- the test:spec-min and test:spec-main were testing the naive implementations
- we don't currently implement both mainnet and minimal spec tests for every test for our fast impl. In fact, we usually only implement one or the other, inconsistently

As it stands, adding/editing spec tests is a very verbose and time consuming task. As an enhancement, it would be great to refactor the spec tests to make it more clearly correct and comprehensive.